projects/js-packages/publicize-components/changelog/update-SOCIAL-208-sidebar-updates

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Social: Improving sidebar by making titles more descriptive and combining sections.
```

Comment:
We update these during the release, may be it's better to fix them now.

We use imperative sentences for changelog entries.


```suggestion
Social: Improve the sidebar by making titles more descriptive and combining sections.
```